Package.swift

```diff
@@ -1,30 +1,35 @@
-// swift-tools-version:5.7
+// swift-tools-version:5.10
 
 import PackageDescription
 
 let package = Package(
     name: "URLSessionDecodable",
     platforms: [
-        .macOS(.v10_13),
-        .iOS(.v11),
-        .tvOS(.v11),
+        .macOS(.v13),
+        .iOS(.v12),
+        .tvOS(.v12),
```

Comment:
why not V15?

Reply:
Just bumped it to the oldest recommended version. We won't get any benefit from v15 here, v16 would make a difference